src/update/NoUpdatesAvailableDialog.h

```diff
@@ -0,0 +1,26 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  NoUpdatesAvailableDialog.h
+
+  Anton Gerasimov
+
+**********************************************************************/
+
+#pragma once
+
+	
+#include "widgets/wxPanelWrapper.h"
+
+/// Information dialog about no updates available, that allows to navigate to settings quickly
```

Comment:
Doxygen comment here, that's good.

Doxygen recognizes a few different formattings, and we are not wholly consistent but I prefer `//!` or `/*! ... */` comments